frontend/pages/_document.js

```diff
@@ -1,6 +1,21 @@
 import Document, { Html, Head, NextScript, Main } from 'next/document';
+import { ServerStyleSheet } from 'styled-components';
 
 class MyDocument extends Document {
+  static getInitialProps(ham) {
+    const h = ham.renderPage
+    const sheet = new ServerStyleSheet();
+    const page = h(
+      function (App) { 
+        return function (props) { 
+          const a = <App {...props} />
+          return sheet.collectStyles(a)
```

Comment:
The return here is the return from the collectStyles method. Set it to a variable and return.